### ISSUE TITLE
[Tsumino] Crashes upon loading most titles, can't download either

### ISSUE THREAD

**Issue by Chromagram** [xOnlyFadi/xonlyfadi-extensions#2]
**Describe the bug**
It doesn't happen to all titles, but many titles you click to read on the new Tsumino source crashes Paperback as soon as you open them.

**To Reproduce**
1. Open any random title under the Tsumino source.
2. Attempt to open a chapter. Sometimes, it is worth trying multiple chapters as one may work fine while another crashes Paperback.

**Expected behavior**
Titles should load the appropriate manga when selected instead of crashing. Just to make sure, I loaded the corresponding manga from the Tsumino site directly, and it has no errors from the site itself.

**Screenshots/Video**
N/A, just crashes the whole app with no popups.

**App version**
- v0.7-r46

**Source WITH source version**
- Tsumino 1.0.2

**Additional context**
If it helps, I have about 5 other sources I use that have no major full-crash level issues, just as a control sample. If you try and download any title that crashes Paperback, it will also immediately fail the download.

**Comment by xOnlyFadi** [comment]
I tested on my side the does not crash when opening an multi chapter and it loads everything needed without any crashes

**Comment by Chromagram** [comment]
Is there anything I can particularly do to test further on my specific device? I'm not exactly running a beta or Paperback or iPadOS or anything either, so I'm not sure what I can do with "it's working on my side".

Addendum: I'll also note that Nana from the same repository has had 0 issues for me, it's just Tsumino causing the issues.

**Comment by xOnlyFadi** [comment]
Which version are you of paperback app store or altstore or testflight?

**Comment by Chromagram** [comment]
Paperback version is v0.7-r46, official App Store build. No TestFlight, no AltStore.

**Comment by xOnlyFadi** [comment]
Im going to test it on the app store version and see if it crashes because currently for me nothing is happening and any series i tried dosen't crash the app.

**Comment by xOnlyFadi** [comment]
can you give me a series that you tried with and it crashed?

**Comment by Chromagram** [comment]
Sure thing. One title I noticed is under just the title XXX, artist Makiba. Boot it up, instant crash for me.

**Comment by xOnlyFadi** [comment]
just tested on phone my phone and ipad still not crashes with both version beta (Testflight) or Stable (App Store)

**Comment by Chromagram** [comment]
Are there any Experimental Features I should or shouldn't have on? Just to make sure the settings are the same

**Comment by xOnlyFadi** [comment]
do you have any Experimental Features enabled?

**Comment by Chromagram** [comment]
At the moment, I do not. I could try ticking any if needed though.

**Comment by xOnlyFadi** [comment]
the problem might be on your side because none of the people that use my extension don't have that problem causing the app to crash.

**Comment by Chromagram** [comment]
I'll fiddle with the 3 experimental features and update if anything helps. Thanks for doing the extensive check.

**Comment by Chromagram** [comment]
Just a little update to close the thread:

I didn't have any experimental features turned on when I posted the issue. After tinkering for a while to fix this, I found the only bizarre thing that worked was to toggle on Async Sources and Legacy Reader, close the app, and then toggle those back off and reset the app again. Since then, it hasn't caused the issue. Really bizarre, but I'll take it. Thanks again for deep diving with some additional tests to confirm the source as a whole was working.
